### PR TITLE
[ServiceBus]: Api version 2023-01-01-preview to support data disaster recovery 

### PR DESCRIPTION
**Pull request by schaudhari6254888**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by rkmanda — DISMISSED** [review]
:shipit:

File: specification/servicebus/resource-manager/readme.md [hunk 1]
```diff
@@ -238,5 +257,36 @@ See configuration in [readme.go.md](./readme.go.md)
...
+    reason: Not a mandatory check
+  - suppress: PutResponseCodes
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: TrackedResourcePatchOperation
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
+  - suppress: SystemDataDefinitionsCommonTypes
+    from: AuthorizationRules.json
+    reason: Not a mandatory check
```
Comment (rkmanda) [inline specification/servicebus/resource-manager/readme.md:289]: These are all mandatory checks. Please provide proper justifications.

**Review by rkmanda — DISMISSED** [review]
:clock1:

File: specification/servicebus/resource-manager/readme.md (hunk 1 above)
Comment (schaudhari6254888) [reply specification/servicebus/resource-manager/readme.md]: Hey @rkmanda , I have given justification for suppression 
can you pls, approve now.

**Review by madhura-np — APPROVED** [review]

**Pull request merged**